Makefile

```diff
@@ -4,6 +4,14 @@ SHELL=/bin/sh
 
 default: it
 
+auto_destdir.c: \
+auto-str conf-destdir
```

Comment:
I fear this doesn't really make anything better for packagers. They like real staging directories, passed by DESTDIR environment variable.

Reply:
I agree that third-party software supporting `${DESTDIR}` is very friendly to packagers. As a packager, I can also attest that this change (and similar ones I made in ucspi-tcp, djbdns, et al.) solved the staging problem just as well as `${DESTDIR}` would have, and I think less invasively.

Reply:
It just means the Makefile is worse than the average. Probably just it's age.